my-app/src/ReservationForm.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import "./ReservationForm.css"
+function ReservationForm({mealTitle, mealPrice, maxReservations, mealId}) {
+    const [clicked, setClicked] = useState(false)
+    const [reservation, setReservation]= useState([])
+    const [inputsValues, setInputValues]= useState(
+        {   id: mealId,
+            phoneNumber: "",
+            name: "",
+            email: "",
+            guests: ""
+        })
+
+    useEffect(()=> {
+        fetch(`http://localhost:5000/api/reservations/${mealId}`)
+            .then((response)=> response.json())
+            .then((data)=> setReservation(data))
+    }, [])
+    const numberOfGuests = reservation.map((reservation)=> reservation.number_of_guests);
+    const availbleSeats = maxReservations - numberOfGuests;
+    console.log("number of guets", numberOfGuests );
+    console.log("type of number of guets", typeof numberOfGuests );
+    console.log("max resev", maxReservations)
+    const handelTelNumber = (e) => {
+        setInputValues({...inputsValues, phoneNumber: e.target.value })
+    }
+    const handelName = (e) => {
+        setInputValues({...inputsValues, name: e.target.value })
+    }
+    const handelEmail = (e) => {
+        setInputValues({...inputsValues, email: e.target.value})
+    }
+    const handelNumberOfGuests = (e) => {
+        setInputValues({...inputsValues, guests: e.target.value})
+    }
+    const handelClick = () => {
```

Comment:
I think `handle` is spelled wrong here :) it works but would be a nice small fix